homeassistant/components/greeneye_monitor/config_flow.py

```diff
@@ -0,0 +1,33 @@
+"""Config flows for greeneye_monitor."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.const import CONF_PORT
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN
+
+
+class GreeneyeMonitorConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for greeneye_monitor."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> data_entry_flow.FlowResult:
+        """Create a config entry from UI."""
+        if await self.async_set_unique_id(DOMAIN):
```

Comment:
Why is only one instance supported? Can't you have multiple instances on multiple ports?